routes/Users.js

```diff
@@ -37,11 +39,87 @@ users.post('/register', (req, res) => {
                         })
                 })
             } else {
-                res.json({ error: "User already exists" })
+
+                res.status(403).json({ error: "User already exists" })
             }
         })
         .catch(err => {
             res.send('error: ' + err)
         })
 });
+
+
+users.post('/login', (req, res) => {
+    User.findOne({
+        where: {
+            email: req.body.email
+        }
+    })
+        .then(user => {
+            if (user) {
+                if (bcrypt.compareSync(req.body.password, user.password)) {
```

Comment:
Another important clean-code practice is to extract and abstract. To avoid 'tall' functions you would like to extract self-contained parts into smaller functions. In this instance you could try something like:

```
function findUser(email) {
    return User.findOne({
        where: {
            email: email
        }
    })
}

function isValidPassword(requestPassword, savedPassword) {
    bcrypt.compareSync(requestPassword, savedPassword)
}

function getToken(user) {
    return jwt.sign(user.dataValues, process.env.SECRET_KEY, {
        expiresIn: 1440
    });
}

function verifyPassword(requestPassword, user) {
    if (isValidPassword(requestPassword, user.password)) {
        res.send(getToken(user));
    } else {
        res.status(403).json({ error: 'Incorrect password'});
    }
}

users.post('/login', (req, res) => {
    findUser(req.body.email).then(user => {
            if (user) {
                verifyPassword(req.body.password, user);
            } else {
                res.status(404).json({ error: 'User does not exist' });
            }
        })
        .catch(err => {
            res.status(400).json({ error: err })
        })
});
```

You could then reuse the smaller functions, for example logic that is now inside `findUser` is also used in other route handlers in this file. The code is also easier to reason about, you could read it like a story and if you need more details about a specific part, you then analyze the responsible function.

Also, remember to use proper HTTP status codes. So, when a user is not found, you would like the code to inform about that with 404 - not found - or when a password is incorrect with 403 - forbidden.